docs/_docs/energy.md

```diff
@@ -202,45 +202,48 @@ If `shift=false`, the potential is left unshifted and any given cutoff is ignore
+\tilde{u}^{(z\mu)}\_{ij}(\bar{r}) = -\frac{ez\_i\left(\mu\_j\cdot \hat{r}\right) }{|\bar{r}|^2} \left( \mathcal{S}(q) - q\mathcal{S}^{\prime}(q) \right)
 $$
 
-where $\hat{\bf r} = {\bf r}/|{\bf r}|$, and the dipole-dipole interaction by
+where $\hat{r} = \bar{r}/|\bar{r}|$, and the dipole-dipole interaction by
 
 $$
-\tilde{u}^{\mu\mu}\_{ij}({\bf r}) = -\left ( \frac{3 ( \boldsymbol{\mu}\_i \cdot \hat{\bf r} ) \left(\boldsymbol{\mu}\_j\cdot\hat{\bf r}\right) - \boldsymbol{\mu}\_i\cdot\boldsymbol{\mu}\_j }{|{\bf r}|^3}\right) \left( \mathcal{S}(q) - q\mathcal{S}^{\prime}(q)  + \frac{q^2}{3}\mathcal{S}^{\prime\prime}(q) \right) - \frac{\left(\boldsymbol{\mu}\_i\cdot\boldsymbol{\mu}\_j\right)}{|{\bf r}|^3}\frac{q^2}{3}\mathcal{S}^{\prime\prime}(q).
+\tilde{u}^{\mu\mu}\_{ij}(\bar{r}) = -\left ( \frac{3 ( \boldsymbol{\mu}\_i \cdot \hat{r} ) \left(\boldsymbol{\mu}\_j\cdot\hat{r}\right) - \boldsymbol{\mu}\_i\cdot\boldsymbol{\mu}\_j }{|\bar{r}|^3}\right) \left( \mathcal{S}(q) - q\mathcal{S}^{\prime}(q)  + \frac{q^2}{3}\mathcal{S}^{\prime\prime}(q) \right) - \frac{\left(\boldsymbol{\mu}\_i\cdot\boldsymbol{\mu}\_j\right)}{|\bar{r}|^3}\frac{q^2}{3}\mathcal{S}^{\prime\prime}(q).
 $$
 
+**Warning:**
+
+ - The `zahn` and `fennell` approaches have undefined dipolar self-energies (see next section) and are therefore not recommended for dipolar systems. 
+
 
 ### Self-energies
 
 When using `coulomb` or `multipole`, an electrostatic self-energy term is automatically
 added to the Hamiltonian. The monopole and dipole contributions are evaluated according to
 
 $$
-U\_{self} = -\frac{1}{2}\sum\_i^N\sum\_{\ast\in\{z,\mu\}} \lim\_{|{\bf r}\_{ii}|\to 0}\left( u^{(\ast\ast)}\_{ii}({\bf r}\_{ii})
-- \tilde{u}^{(\ast\ast)}\_{ii}({\bf r}\_{ii}) \right )
+U\_{self} = -\frac{1}{2}\sum\_i^N\sum\_{\ast\in\{z,\mu\}} \lim\_{|\bar{r}\_{ii}|\to 0}\left( u^{(\ast\ast)}\_{ii}(\bar{r}\_{ii})
+- \tilde{u}^{(\ast\ast)}\_{ii}(\bar{r}\_{ii}) \right )
 $$
 
 where no tilde indicates that $\mathcal{S}(q)\equiv 1$ for any $q$.
 
 
 ### Ewald Summation
 
-If type is `ewald`, terms from reciprocal space; surface energies; and
-self energies are automatically added to the Hamiltonian, activating additional keywords:
+If type is `ewald`, terms from reciprocal space and surface energies are automatically added (in addition to the previously mentioned self- and real space-energy) to the Hamiltonian which activates the additional keywords:
 
 `type=ewald`          | Description
 --------------------- | ---------------------------------------------------------------------
-`kcutoff`             | Reciprocal-space cutoff
+`ncutoff`             | Reciprocal-space cutoff (unitless)
 `epss=0`              | Dielectric constant of surroundings, $\varepsilon_{surf}$ (0=tinfoil)
 `ewaldscheme=PBC`     | Periodic (`PBC`) or isotropic periodic ([`IPBC`](http://doi.org/css8)) boundary conditions
 `spherical_sum=true`  | Spherical/ellipsoidal summation in reciprocal space; cubic if `false`.
```

Comment:
`spherical_sum` is missing from `docs/schema.yml`; could you add it so that the input is properly checked?

Reply:
(just search for Ewald in the file and you'll see where to insert it)